libraries/core/src/main/java/com/chesire/nekome/core/settings/ApplicationSettings.kt

```diff
@@ -49,6 +51,23 @@ class ApplicationSettings @Inject constructor(
             return status
         }
 
+    /**
+     * Gets the default [HomeScreenOption] the home screen that shows after login
+     */
+    val defaultHomeScreen: HomeScreenOptions
+        get() {
+            val index = requireNotNull(
+                preferences.getString(
+                    _defaultHomeScreen,
+                    HomeScreenOptions.Anime.index.toString()
+                )
+            ) {
+                "Preferences defaultSeriesState returned null, with supplied default value"
```

Comment:
Error statement here should mention the `defaultHomeScreen` instead of `defaultSeriesState`